rules/no-bitwise-trunc.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'no-bitwise-trunc';
+const messages = {
+	[MESSAGE_ID]: 'Use `Math.trunc` instead of `| 0`.'
+};
+
+const binaryExpressionSelector = [
+	'BinaryExpression',
+	'[operator="|"]',
+	'[left.type=/(Literal|Identifier)/]',
+	'[right.type="Literal"]',
+	'[right.value=0]'
```

Comment:
I'm not sure. But I think `| "0"` will get selected, we can use `[left.raw=0]`, just ignore `| 0.0`.

Reply:
Actually `| "0"` behave the exact same as `0` (it will truncate the number, and it has the same ""bug""). I think it should be OK if it is also auto-fixed. This will also allow `0.0` to be taken into account.

Reply:
will `null` get selected? I'm not sure.

Reply:
![image](https://user-images.githubusercontent.com/34779161/95014536-de7b4f80-0647-11eb-8c42-46f9be49a537.png)
Idk i'm lost, it works with *everything*

Reply:
I prefer only check `| 0`